### ISSUE TITLE
Using Viz as a standalone React component doesn't work out of the box

### ISSUE THREAD

**Issue by tynandebold** [kedro-org/kedro-viz#738]
## Description
Using Kedro-Viz as a standalone React component doesn't work out of the box.

## Context
I'm trying to use Viz as a standalone component in my Create React App project. I'm getting some errors, and when those are remedied, I don't see the anything in the browser.

## Steps to Reproduce

1. Create a new project with Create React App: `npx create-react-app kedro-viz-standalone-component`
2. `cd` into the project and then install the the Viz dependency:  `npm install @quantumblack/kedro-viz --save`
3. Add a file with some dummy Kedro data ([data example](https://github.com/kedro-org/kedro-viz/blob/main/src/utils/data/spaceflights.mock.json))
4. Update `App.js` to include the Viz component and the data:
```jsx
import "./App.css";
import KedroViz from "@quantumblack/kedro-viz";
import { demo } from "./demo";

function App() {
  return (
    <div className="App">
      <KedroViz data={demo} />
    </div>
  );
}

export default App;
```
5. Run the app with `npm  start`.
6. Observe the compile errors and subsequently the blank webpage.

## Expected Result
The app should run without error and I should see Kedro Viz running locally.

## Actual Result
I received some dependency errors: 
![Screen Shot 2022-02-10 at 10 49 25](https://user-images.githubusercontent.com/16869061/153396547-65c2fd92-2c9f-4a53-a84e-1b5b4cace4af.png)

And when I fixed those and ran the app again, nothing showed up, thought the DOM elements exist:
![Screen Shot 2022-02-10 at 12 16 31](https://user-images.githubusercontent.com/16869061/153407146-1a0e9865-ee98-48c9-b31a-828dd1cb88ea.png)

## Your Environment
Include as many relevant details as possible about the environment you experienced the bug in:

* Web browser system and version: Google Chrome Version 98.0.4758.80 (Official Build) (x86_64)
* Operating system and version: MacOS 11.6.2 (20G314)
* NodeJS version used (if relevant): 14.17.0
* Kedro version used (if relevant): 4.3.1

## Checklist
- [x] Include labels so that we can categorise your issue

**Comment by studioswong** [comment]
@rashidakanchwala just saw that you had picked up this issue - I had started looking into this alongside issue #758 as this needs to be fixed in order to ship #758 , and to avoid duplicate efforts I can assign this to myself, if that's ok with you?

**Comment by studioswong** [comment]
Thanks for reporting this - in digging deeper into this issue, it looks like the error is likely due to the reference for the `faker` npm package has been updated to `@faker-js/faker`( see[ here](https://www.npmjs.com/package/@faker-js/faker). 

We will update the package reference to `@faker-js/faker` and publish a new npm package in the latest release.

**Comment by tynandebold** [comment]
Nice, thanks a lot for this.

And are you not seeing the blank page when Viz is loaded, as I was? I don't see any mention of that in your comment above.

**Comment by studioswong** [comment]
yea about that - I am slightly confused with what you mean as fixing that error locally for a third party import. So that I can replicate the blank page that you mentioned, can you please described the steps that you did when you mentioned 'fixed those and ran the app again'?

**Comment by tynandebold** [comment]
That line refers to installing the missing dependencies. So when I fixed the errors related to that and ran the app again, I saw a blank white page without anything on it.

**Comment by studioswong** [comment]
Update: @tynandebold and I had a catch up with regards to the above issue, and we have identified that the blank screen is a separate issue caused by the lack of a parent wrapping element that specifies the height of the container for the `Kedro-Viz` component. 

To bridge the gap in avoiding this to happen for our external users, I had updated our readme as part of the related PR to mention the need for a parent container element with a specified height, as seen in [this commit](https://github.com/kedro-org/kedro-viz/pull/784/commits/b146bee538e3111da5998fba4accc9b879f6bef9) and below:

<img width="1035" alt="image" src="https://user-images.githubusercontent.com/27775658/158201150-bfc7f4e1-969f-4047-b8eb-7b2d14813740.png">

**Comment by tynandebold** [comment]
> Update: @tynandebold and I had a catch up with regards to the above issue, and we have identified that the blank screen is a separate issue caused by the lack of a parent wrapping element that specifies the height of the container for the `Kedro-Viz` component.
> 
> To bridge the gap in avoiding this to happen for our external users, I had updated our readme as part of the related PR to mention the need for a parent container element with a specified height, as seen in [this commit](https://github.com/kedro-org/kedro-viz/pull/784/commits/b146bee538e3111da5998fba4accc9b879f6bef9) and below:
> 
> <img alt="image" width="1035" src="https://user-images.githubusercontent.com/27775658/158201150-bfc7f4e1-969f-4047-b8eb-7b2d14813740.png">

Looks great.

One nit: can you put the `<KedroViz data={json} />` example on one line? It would be formatted as such in our editors. I would also say that we should reference Viz like this (`<KedroViz />`) when we're taking about it in its standalone component form.

**Comment by tynandebold** [comment]
Closed by #758
